### DISCUSSION TITLE
Make chat censor toggleable

### DISCUSSION THREAD

**Discussion by A6sT** [Quaver/Quaver#4788] [Feature Request]
### Is your feature request related to a problem? Please describe.

As of today, the censor feature is enabled by default in chats (be it in the general chat or any multiplayer room), and there's no way to disable it. The main issue I have with this is that if you end up speaking another language (with some friend in a multiplayer lobby for example), a lot of words / sentences will get censored, which shouldn't happend.

### Describe the solution you'd like

A solution to this issue would be to add a new option to toggle chat censorring. From what I've understood, this would require a bit of server rewrite as censor is handled by the server.

### Describe alternatives you've considered

Alternatives (that are clearly not ideal) would be to make censor based on the country of the player, but this sounds like a lot of works and efforts, which isn't really worth it, unless there are no other way around it

**Comment by AiAe** [comment]
@Swan do you think it would be better to move the censoring to client? So it can be disabled.

**Comment by Swan** [comment]
Sure

**Comment by EpiCreeper** [comment]
this change i* **celent
